[2025-03-23T18:54:46.818+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-23T18:54:46.846+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-22T08:00:00+00:00 [queued]>
[2025-03-23T18:54:46.857+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-22T08:00:00+00:00 [queued]>
[2025-03-23T18:54:46.858+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-23T18:54:46.876+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): move_cloud_clientes> on 2025-03-22 08:00:00+00:00
[2025-03-23T18:54:46.884+0000] {standard_task_runner.py:72} INFO - Started process 1149 to run task
[2025-03-23T18:54:46.889+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'schedule', 'move_cloud_clientes', 'scheduled__2025-03-22T08:00:00+00:00', '--job-id', '1894', '--raw', '--subdir', 'DAGS_FOLDER/dag_master.py', '--cfg-path', '/tmp/tmpbil2a89t']
[2025-03-23T18:54:46.894+0000] {standard_task_runner.py:105} INFO - Job 1894: Subtask move_cloud_clientes
[2025-03-23T18:54:46.921+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-23T18:54:46.979+0000] {task_command.py:467} INFO - Running <TaskInstance: schedule.move_cloud_clientes scheduled__2025-03-22T08:00:00+00:00 [running]> on host b612e1a2c49c
[2025-03-23T18:54:46.981+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-23T18:54:47.076+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule' AIRFLOW_CTX_TASK_ID='move_cloud_clientes' AIRFLOW_CTX_EXECUTION_DATE='2025-03-22T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-22T08:00:00+00:00'
[2025-03-23T18:54:47.079+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-23T18:54:47.090+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-23T18:54:47.096+0000] {subprocess.py:88} INFO - Running command: ['/bin/bash', '-c', 'docker exec linux_python /bin/bash -c \'/workspace/scripts/run_move_file.sh /path/to/move_cloud/clientes/clientes.csv table_ingestion_files/clientes/20250323/clientes_1742756086.csv && echo "Script Executado com Sucesso!"\'']
[2025-03-23T18:54:47.107+0000] {subprocess.py:99} INFO - Output:
[2025-03-23T18:54:47.784+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,780 - INFO - Iniciando o upload do arquivo '/path/to/move_cloud/clientes/clientes.csv' para o blob 'table_ingestion_files/clientes/20250323/clientes_1742756086.csv' no container 'filedataimport'.
[2025-03-23T18:54:47.792+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,780 - INFO - Blob URL gerada: https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientes/20250323/clientes_1742756086.csv?sp=cw&st=2024-10-20T02:42:12Z&se=2026-10-20T10:42:12Z&spr=https&sv=2022-11-02&sr=c&sig=MaoRwZWwtKEx5vDafwkbeHkx1Rir6bWRYy0FB3OOs58%3D
[2025-03-23T18:54:47.794+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,783 - INFO - Enviando o arquivo '/path/to/move_cloud/clientes/clientes.csv' para o Azure Blob Storage.
[2025-03-23T18:54:47.795+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:47,786 - INFO - Request URL: 'https://stagedatamaster.blob.core.windows.net/filedataimport/table_ingestion_files/clientes/20250323/clientes_1742756086.csv?sp=REDACTED&st=REDACTED&se=REDACTED&spr=REDACTED&sv=REDACTED&sr=REDACTED&sig=REDACTED'
[2025-03-23T18:54:47.797+0000] {subprocess.py:106} INFO - Request method: 'PUT'
[2025-03-23T18:54:47.798+0000] {subprocess.py:106} INFO - Request headers:
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'Content-Length': '558955'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'x-ms-blob-type': 'REDACTED'
[2025-03-23T18:54:47.799+0000] {subprocess.py:106} INFO -     'If-None-Match': '*'
[2025-03-23T18:54:47.800+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-03-23T18:54:47.800+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/octet-stream'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO -     'Accept': 'application/xml'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO -     'User-Agent': 'azsdk-python-storage-blob/12.24.1 Python/3.12.9 (Linux-6.10.14-linuxkit-aarch64-with-glibc2.36)'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO -     'x-ms-date': 'REDACTED'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': '4aee49ac-0818-11f0-89f6-0242ac120004'
[2025-03-23T18:54:47.801+0000] {subprocess.py:106} INFO - A body is sent with the request
[2025-03-23T18:54:48.060+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,057 - INFO - Response status: 409
[2025-03-23T18:54:48.062+0000] {subprocess.py:106} INFO - Response headers:
[2025-03-23T18:54:48.062+0000] {subprocess.py:106} INFO -     'Content-Length': '220'
[2025-03-23T18:54:48.062+0000] {subprocess.py:106} INFO -     'Content-Type': 'application/xml'
[2025-03-23T18:54:48.063+0000] {subprocess.py:106} INFO -     'Server': 'Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0'
[2025-03-23T18:54:48.063+0000] {subprocess.py:106} INFO -     'x-ms-request-id': 'c28ae44e-601e-006b-1c25-9c697a000000'
[2025-03-23T18:54:48.064+0000] {subprocess.py:106} INFO -     'x-ms-client-request-id': '4aee49ac-0818-11f0-89f6-0242ac120004'
[2025-03-23T18:54:48.066+0000] {subprocess.py:106} INFO -     'x-ms-version': 'REDACTED'
[2025-03-23T18:54:48.068+0000] {subprocess.py:106} INFO -     'x-ms-error-code': 'BlobAlreadyExists'
[2025-03-23T18:54:48.069+0000] {subprocess.py:106} INFO -     'Date': 'Sun, 23 Mar 2025 18:54:47 GMT'
[2025-03-23T18:54:48.076+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,063 - ERROR - Erro ao enviar o arquivo '/path/to/move_cloud/clientes/clientes.csv': The specified blob already exists.
[2025-03-23T18:54:48.076+0000] {subprocess.py:106} INFO - RequestId:c28ae44e-601e-006b-1c25-9c697a000000
[2025-03-23T18:54:48.076+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9787332Z
[2025-03-23T18:54:48.077+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-03-23T18:54:48.077+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-03-23T18:54:48.078+0000] {subprocess.py:106} INFO - RequestId:c28ae44e-601e-006b-1c25-9c697a000000
[2025-03-23T18:54:48.082+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9787332Z</Message></Error>
[2025-03-23T18:54:48.084+0000] {subprocess.py:106} INFO - 2025-03-23 18:54:48,063 - ERROR - Falha no processo de upload: The specified blob already exists.
[2025-03-23T18:54:48.086+0000] {subprocess.py:106} INFO - RequestId:c28ae44e-601e-006b-1c25-9c697a000000
[2025-03-23T18:54:48.086+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9787332Z
[2025-03-23T18:54:48.087+0000] {subprocess.py:106} INFO - ErrorCode:BlobAlreadyExists
[2025-03-23T18:54:48.088+0000] {subprocess.py:106} INFO - Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobAlreadyExists</Code><Message>The specified blob already exists.
[2025-03-23T18:54:48.089+0000] {subprocess.py:106} INFO - RequestId:c28ae44e-601e-006b-1c25-9c697a000000
[2025-03-23T18:54:48.089+0000] {subprocess.py:106} INFO - Time:2025-03-23T18:54:47.9787332Z</Message></Error>
[2025-03-23T18:54:48.132+0000] {subprocess.py:106} INFO - Script Executado com Sucesso!
[2025-03-23T18:54:48.145+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-23T18:54:48.215+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-23T18:54:48.219+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=schedule, task_id=move_cloud_clientes, run_id=scheduled__2025-03-22T08:00:00+00:00, execution_date=20250322T080000, start_date=20250323T185446, end_date=20250323T185448
[2025-03-23T18:54:48.268+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-23T18:54:48.305+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
